OpenMarket/OpenMarket/Controller/ProductListViewController.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  ProductListViewController.swift
+//  Created by Lingo, Quokka
+// 
+
+import UIKit
+
+fileprivate enum SegmentIndex: Int {
+  case list = 0
+  case grid
+}
+
+final class ProductListViewController: UIViewController {
+  private enum Constant {
+    static let segmentWidth = 80.0
+  }
+  
+  private let networkService = APINetworkService(urlSession: URLSession.shared)
+  private var productList = [Product]() {
+    didSet {
+      DispatchQueue.main.async {
+        self.collectionView.reloadData()
+      }
+    }
+  }
+  
+  private lazy var addProductButton = UIBarButtonItem(
+    barButtonSystemItem: .add,
+    target: self,
+    action: #selector(didTapAddProductButton))
+
+  private lazy var segmentControl: UISegmentedControl = {
+    let segment = UISegmentedControl(items: ["LIST", "GRID"])
+    segment.setWidth(Constant.segmentWidth, forSegmentAt: .zero)
+    segment.setWidth(Constant.segmentWidth, forSegmentAt: 1)
+    segment.selectedSegmentIndex = .zero
```

Comment:
[IMO] 여기에 쓰이는 프로퍼티값들을 SegmentIndex 내부에 프로퍼티를 만들어두고 그걸 쓰는건 어떨가요? ㅎㅎ (데이터 싱크도 맞고)
```suggestion
    let segment = UISegmentedControl(items: [SegmentIndex.list.title, SegmentIndex.grid.title])
    segment.setWidth(Constant.segmentWidth, forSegmentAt: SegmentIndex.list.rawValue)
    segment.setWidth(Constant.segmentWidth, forSegmentAt: SegmentIndex.grid.rawValue)
    segment.selectedSegmentIndex = SegmentIndex.list.rawValue
```